test/index.js

```diff
@@ -170,6 +171,7 @@ test('when preloaded with -r, automatically logs all debug calls with log level
   module.filename = filename
   module.parent = parent
 })
+*/
```

Comment:
hey @davidmarkclements this test was broken before running. Not sure what happens.